homeassistant/components/nobo_hub/strings.json

```diff
@@ -0,0 +1,66 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Start by entering the complete serial number (12 digits) of the Nobø Ecohub you want to add. If you have more than one hub, and they have the same serial prefix, you must also specify the IP address. If any devices were automatically found on your network, they are shown below.\n\nIf you cannot see your device or experience any issues, try specifying the device IP address.\n\n{devices}",
```

Comment:
Did you consider making the list of discovered devices selectable during the config flow rather than making the user re-enter the text here? I realize you also want an option for manual entry in case discovery fails, it just seems more user friendly to make auto discovered devices not require any text input.

Reply:
Auto discovered devices still need to get the 3 last digits added to their serial number. I guess this is Nobø's security mechanism. Also, in the assumed case you have two hubs on your local network (I don't know why anybody would have that though), and they have the same serial prefix, we're not able to distinguish the two hubs, so then we also need the ip address even though the hub is auto discovered.

Reply:
This is fixed in the new config flow.